net/tlsdial/tlsdial.go

```diff
@@ -30,6 +42,52 @@ func Config(host string, base *tls.Config) *tls.Config {
 	}
 	conf.ServerName = host
 
+	if n := sslKeyLogFile; n != "" {
+		f, err := os.OpenFile(n, os.O_CREATE|os.O_APPEND|os.O_WRONLY, 0600)
+		if err != nil {
+			log.Fatal(err)
+		}
+		log.Printf("WARNING: writing to SSLKEYLOGFILE %v", n)
+		conf.KeyLogWriter = f
+	}
+
+	if conf.InsecureSkipVerify {
+		panic("unexpected base.InsecureSkipVerify")
+	}
+	if conf.VerifyConnection != nil {
+		panic("unexpected base.VerifyConnection")
+	}
+	conf.InsecureSkipVerify = true
```

Comment:
maybe add to the line `// security provided by VerifyConnection below`

Reply:
Done. I add the same comment below but didn't copy it. 

At some point I should unify that code, but this PR didn't feel like the time for it.

Reply:
PTAL